.gitignore

```diff
@@ -1,6 +1,7 @@
 .settings
 .buildpath
 .project
+.idea
```

Comment:
IDE's have been removed since latest update, you should configure this in your global `.gitignore`.